hasura/migrations/1606365023960_create_table_public_Map/up.sql

```diff
@@ -0,0 +1,2 @@
+CREATE EXTENSION IF NOT EXISTS pgcrypto;
+CREATE TABLE "public"."Map"("id" uuid NOT NULL DEFAULT gen_random_uuid(), "player_id" uuid NOT NULL, "name" text NOT NULL, "data" text NOT NULL, PRIMARY KEY ("id") , FOREIGN KEY ("player_id") REFERENCES "public"."Player"("id") ON UPDATE restrict ON DELETE restrict, UNIQUE ("id"));
```

Comment:
I would call it something like "owner_id" or "author_id" instead so it's clear who this player is in relation to the map

Reply:
Agreed! Right now, I'm storing everything using `data` as a text blob.
I'm unsure if we want to integrate it as `MapItem`. I don't think there are any real benefits since everything is rendered interactively client side anyway.

In my opinion, if this is good as is. Let's change it to author_id. I will be merging the MetaMaps core codebase soon.